openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -261,8 +261,8 @@ def process(self, instance):
             "ext": capture_preset["Codec"]["compression"],
             "files": collected_files,
             "stagingDir": stagingdir,
-            "frameStart": start,
-            "frameEnd": end,
+            "frameStart": int(start),
```

Comment:
Would things break( I could only see `preset` using it) if this were to happen at line 64 with some `try..except` ?

Reply:
I don't know... I know it breaks things if it's float on representation.